test_setup.py

```
@@ -226,7 +229,9 @@ def test_web_application():
 def main():
     """Run all tests"""
     print_header("🚀 Real-Time Stock Market ML Project Setup Test")
-    print_info(f"Test started at: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}")
+    print_info(
+        f"Test started at: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}"
```

Ruff (0.14.14)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)